element_array_ephys/ephys.py

```diff
@@ -187,12 +190,13 @@ def make(self, key):
                           'acq_software': acq_software,
                           'sampling_rate': spikeglx_meta.meta['imSampRate']})
 
-            root_dir = find_root_directory(get_ephys_root_data_dir(), meta_filepath)
+            root_dir = find_root_directory(get_ephys_root_data_dir(), 
+                                           meta_filepath)
             self.EphysFile.insert1({
                 **key,
                 'file_path': meta_filepath.relative_to(root_dir).as_posix()})
         elif acq_software == 'Open Ephys':
-            dataset = openephys.OpenEphys(sess_dir)
+            dataset = openephys.OpenEphys(session_dir)
             for serial_number, probe_data in dataset.probes.items():
                 if str(serial_number) == inserted_probe_serial_number:
                     break
```

Comment:
```suggestion
            for serial_number in dataset.probes:
                if str(serial_number) == inserted_probe_serial_number:
                    break
```

Reply:
The variable `probe_data` is used in the [next section](https://github.com/kabilar/element-array-ephys/blob/ce6adf1c6a7409ecd98907f3a3a5c6d50f0c10bb/element_array_ephys/ephys.py#L208-L232) after the `for` loop, so we would need to keep this statement as is.

Reply:
Then I would get `probe_data` after the loop
```suggestion
            for serial_number in dataset.probes:
                if str(serial_number) == inserted_probe_serial_number:
                    probe_data = dataset.probes[serial_number]
                    break
```